src/aibs_informatics_cdk_lib/common/aws/iam_utils.py

```diff
@@ -351,6 +381,28 @@ def s3_policy_statement(
     )
 
 
+def secretsmanager_policy_statement(
+    actions: List[str] = SECRETSMANAGER_READ_ONLY_ACTIONS,
+    sid: str = "SecretsManagerReadOnly",
+    resource_id: str = "*",
+    region: str = None,
+    account: str = None,
```

Comment:
I think for something as sensitive as secrets manager read permissions, maybe it might make sense to have the default account filled in and not be `None`?